WordPress/WordPressUITests/Screens/Editor/FeaturedImageScreen.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import XCTest
+
+class FeaturedImageScreen: BaseScreen {
+
+
+    let navigationBar: XCUIElement
+
+    init() {
+        let app = XCUIApplication()
+        navigationBar = app.navigationBars["Featured Image"]
+        super.init(element: navigationBar )
+    }
+
+    func tapRemoveFeaturedImageButton() {
+        navigationBar.buttons["Remove Featured Image"].tap()
+        app.sheets["Remove this Featured Image?"].scrollViews.otherElements.buttons["Remove"].tap()
```

Comment:
Similar to the navigation bar, it would be good to avoid using localized strings here. For action sheets, we've been identifying them like this (tapping the first button on the action sheet):

```suggestion
        app.sheets.buttons.element(boundBy: 0).tap()
```

Reply:
Done.